plugin/lighthouse/metadata.py

```diff
@@ -847,6 +899,14 @@ def _compute_complexity(self):
         complexity calculation. Not doing so will radically throw off the
         cyclomatic complexity score.
         """
+
+        # Cutter already provides this information, so just fetch it
+        if disassembler.NAME == "CUTTER":
+            try:
+                return int(cutter.cmd('afCc @ ' + str(self.address)))
```

Comment:
I would prefer to avoid inline disassembler casing. We have the disassembler API abstraction explicitly so we don't have to have if/else trees everywhere. It makes the code both more maintainable, and readable.

The exception to that rule is mostly class monkey patching, which is done to ensure performance for hot paths.

I'll evaluate whether it is reasonable to add complexity calculations to the disassembler API, or if it is reasonable to simply use the existing code. Complexity calculations are already pretty fast in lighthouse, but I understand that r2 might offer more 'accurate' scores.